index.html

```diff
@@ -152,7 +155,7 @@ <h4>Featured Speakers</h4>
       </section>
       <section class="desktop" id="partners">
         <div class="section-heading">
-          <h4>Partner</h4>
+          <h4>Partners</h4>
           <hr>
```

Comment:
- [ ] Kindly use the same color for the line under title as in the provided design
### Your design :point_down: 
![Screenshot from 2023-05-25 11-39-31](https://github.com/badger-99/Capstone-Project-1/assets/91142432/72f60886-7543-41a1-b1b3-fa651b2ea4f4)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-05-25 11-39-50](https://github.com/badger-99/Capstone-Project-1/assets/91142432/f6f9831b-b30c-4e3c-a506-aeef1cb70eed)

- [ ] Same thing for the "Featured Speakers" section :point_up_2: